spec/fixtures/files/computacenter/CapUsageReportSample.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
Is this referenced anywhere?

I'm generally not a fan of fixtures because you are forced to look in a completely different file when reading the spec. I prefer it when the test files are created from strings within the spec file itself...

Reply:
agreed - I saved the files there first, intending to use them for specs, then by the time I got to writing the specs, went ahead and did the specs from inline strings. I've deleted the files now.